mhkit/loads/bin_statistics.m

```diff
@@ -1,29 +1,30 @@
-function datast=bin_statistics(data,x,bin_edges)
+function bins=bin_statistics(data,bin_against,bin_edges)
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%   Calculates bin statistics against "x" according to IEC
+%   Bins calculated statistics against data signal (or channel) 
+%   according to IEC TS 62600-3:2020 ED1.
 %    
 % Parameters
 % ------------
 %     data: with handles- data.data and data.time  
 %         data.data contains a vector or matrix containing time-series 
 %         statistics of variables 
 %
-%     x: vector
-%         contains array of variable to bin data against (ie. current speed)
+%     bin_against: vector
+%         Data signal to bin data against (ie. current speed)
 %
 %     bin_edges: vector
-%         contains vector of desired bin edges w/ consistent step size.
+%         Bin edges with consistent step size.
 %
```

Comment:
do you want to add the optional input parameter?
data_signal (list, optional) – List of data signal(s) to bin, default = all data signals